guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -82,7 +100,7 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
 
             // If the clipboard data is a string, render it as text
             if (typeof data.data === 'string')
-                element.value = data.data;
+                element.value = (shouldDisplay) ? data.data : '';
```

Comment:
It looks like this just shows an empty clipboard until the user clicks on the clipboard element, and then the data pops in, yeah? It'd probably be lot more obvious to users if there was some sort of styling to indicate an inactive clipboard, and some indication to the user about what to do if they want to see the clipboard.